src/FINDOLOGIC/Export/Data/Attribute.php

```diff
@@ -53,7 +57,7 @@ public function getKey()
      * @SuppressWarnings(PHPMD.StaticAccess)
      * @inheritdoc
      */
-    public function getDomSubtree(\DOMDocument $document)
+    public function getDomSubtree(\DOMDocument $document): \DOMElement
```

Comment:
Consider `use`ing `DOMElement`.

Reply:
There will be a custom branch for this as soon we merged this into the develop branch. I also created a issue #97 for this.